app/views/plans/_form.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="col-md-offset-2 col-md-8">
       <div class="page-header">
-        <h3>Create a new Plan</h3>
+        <h3><%= @title %></h3>
```

Comment:
This heading will always be empty because `@title` isn't defined anywhere. You should either change it to something static or else define that instance variable on the `#new` method from `PlansController `.